### PR TITLE
ci: update codeowners to reflect GitHub team members

### PR DESCRIPTION
**Pull request by gkalpak**
☝️

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -44,14 +44,10 @@
 #  alxhub - Alex Rickabaugh
 #  AndrewKushnir - Andrew Kushnir
```
Comment (josephperrott) [inline .github/CODEOWNERS]: We should also add `atscott - Andrew Scott`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -127,7 +121,6 @@
 #
 #   - alexeagle
```
Comment (josephperrott) [inline .github/CODEOWNERS]: `alexeagle` should be removed from:
`@angular/framework-global-approvers`
`@angular/tools-cli`
`@angular/fw-integration`
`@angular/fw-docs-packaging`
`@angular/tools-docs-libraries`
`@angular/tools-docs-schematics`

**Review by josephperrott — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (gkalpak) [reply .github/CODEOWNERS]: Removed (and updated those teams in `CODEOWNERS` to match the actual GitHub teams).

**Comment by gkalpak** [comment]
The CI failures are unrelated to this PR (and will be fixed by #32602).

**Review by josephperrott — APPROVED** [review]
LGTM

Thanks! I know this was tedious work to have to manage the Github UI to get all of this info lined up.

**Comment by gkalpak** [comment]
(Rebased on master to make CI happy.)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -99,7 +93,6 @@
 #  Used for approving minor documentation-only changes that don't require engineering review.
 #  (secret team to avoid review requests, it also doesn't inherit from @angular/framework because nested teams can't be secret)
 #
-#   - brandonroberts
 #   - gkalpak
 #   - petebacondarwin
```
Comment (kara) [inline .github/CODEOWNERS]: Let's replace brandonroberts with `kapunahelewong`

**Review by kara — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (gkalpak) [reply .github/CODEOWNERS]: Done.

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: global approval (note: Igor approved these changes over IM)

**Pull request closed without merging**